concepts/05 UI Components/Scheduler/060 Views/010 View Types/050 Agenda View.md

```diff
@@ -1,4 +1,4 @@
-The **agenda** view lists all available appointments grouped by date. This view provides a compact representation of appointments, which is useful when displaying on narrow screens. If an appointment lasts for several days, it is displayed for every day it belongs to. If an appointment lasts all day, it is placed above other appointments of that day.
+The **agenda** view lists all available appointments grouped by date. This view provides a compact representation of appointments, which is useful when displaying on narrow screens. If an appointment lasts for several days, it is displayed for every day it belongs to. If an appointment lasts all day, it is placed above other appointments of that day. The Scheduler does not support grouping resources by multiple data fields in the **agenda** view.
 
 ![Agenda View](/images/UiWidgets/Scheduler_Agenda_View.png)
 
```

Comment:
Please make the following changes below:

Line 9:
"For details on individual views customization, see [this topic](/concepts/05%20UI%20Components/Scheduler/060%20Views/020%20Customize%20Individual%20Views.md '/Documentation/Guide/UI_Components/Scheduler/Views/Customize_Individual_Views/')."

1. Please change "views customization" to "view customization".

2. Instead of "see this topic", please include the title of the topic.

Line 101:
"If you group appointmetns by resources..."

"appointmetns" is mispelled. Please fix.

Reply:
Done

Reply:
Done